lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -523,9 +523,8 @@ class DetailsRenderer {
 
     if (!this._fullPageScreenshot) return element;
 
-    const rect =
-      (item.lhId ? this._fullPageScreenshot.nodes[item.lhId] : null) || item.boundingRect;
-    if (!rect) return element;
+    const rect = item.lhId ? this._fullPageScreenshot.nodes[item.lhId] : item.boundingRect;
```

Comment:
I'm thinking the fallback of `item.boundingRect` doesn't make sense... should we remove?

Reply:
Why doesn't it make sense? It's not obvious to me what the issue is.

Reply:
i think the idea here was to show _something_ in case the node doesn't show in `_fullPageScreenshot.nodes`, which could happen if LH registered an element but then it is deleted by the time the full page screenshot gatherer runs. So the user might get an idea of where the element was. Could that be more confusing than helpful?

Reply:
> i think the idea here was to show _something_ in case the node doesn't show in `_fullPageScreenshot.nodes`, which could happen if LH registered an element but then it is deleted by the time the full page screenshot gatherer runs. So the user might get an idea of where the element was. Could that be more confusing than helpful?

I think you're right. In theory if it's not in `fullPageScreenshot.nodes`, it didn't exist in the DOM anymore when the screenshot was taken, so what's the point in trying to show a spot where it wasn't in the screenshot? That could still be wrong (removed _during_ the screenshot taking so not around when `nodes` are gathered?) but it seems better to miss a few corner case nodes that could have had a screenshot than to show confusing/misleading element screenshots.

Reply:
I see. In that case, I agree as well that it doesn't make sense.